tests/automation/release_notes/test_generate.py

```diff
@@ -1,6 +1,11 @@
 import re
+from io import StringIO
+from unittest import mock
+
+from deepdiff import DeepDiff
```

Comment:
We're going towards using only `import X` please change to that
The only exception is `from mlrun.utils import logger` which is ok (for now)